_config/buttons.yml

```diff
@@ -0,0 +1,7 @@
+---
+Name: buttons
+---
+FormActions:
```

Comment:
I feel like this should be put into a better namespace, or at least something that makes it clearer that we're talking about form actions on the detail form of a grid field.

Also, it's not clear how I would override these yml defaults on a specific gridfield. There should probably be properties of the relevant gridfield component that can be overridden.

Reply:
Agreed. Let's also pin these to a concrete class, rather than against the arbitrary `FormActions`, which is really ambiguous.

```
SilverStripe\Forms\GridField\GridFieldDetailForm_ItemRequest:
  formActions:
    ...
```